doc/plugin_agent_svidstore_sconecas_sessionmanager.md

```diff
@@ -17,10 +17,11 @@ When the SVIDs are updated, the plugin takes care of creating (or updating) sess
 
```

Comment:
at the beginning of the markdown, it is said that the flag used is "-svidStore", but during execution an error occurs in which it is necessary to change it to "-storeSVID"

Reply:
Good catch! Thank you, @amand-io